### PR TITLE
Update parent artifact version

### PR DESCRIPTION
**Pull request by CuriousVini**
- Fixed parent artifacts.
- Updated maven bundle plugin version to solve:https://issues.apache.org/jira/browse/FELIX-4005 

image

### CODE REVIEW

**Review by sreevatsanraman — APPROVED** [review]

File: dre-plugins/pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <!-- properties for script budirectivestep that creates the config files for the artifacts -->
   <widgets.dir>widgets</widgets.dir>
   <docs.dir>docs</docs.dir>
-  <app.parents>system:cdap-data-pipeline[4.2.0-SNAPSHOT,4.4.0-SNAPSHOT),system:cdap-data-streams[4.2.0-SNAPSHOT,4.4.0-SNAPSHOT)</app.parents>
+  <app.parents>system:cdap-data-pipeline[4.2.0-SNAPSHOT,6.0.0-SNAPSHOT),system:cdap-data-streams[4.2.0-SNAPSHOT,6.0.0-SNAPSHOT)</app.parents>
```
Comment (bdmogal) [inline dre-plugins/pom.xml]: should we change this to use cdap-maven-plugin?

File: dre-plugins/pom.xml [hunk 2]
```diff
@@ -117,84 +116,21 @@
           </executions>
         </plugin>
         <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-antrun-plugin</artifactId>
-          <version>1.7</version>
+          <groupId>co.cask</groupId>
+          <artifactId>cdap-maven-plugin</artifactId>
+          <version>1.0.0-SNAPSHOT</version>
```
Comment (bdmogal) [inline dre-plugins/pom.xml]: I don't think this is available. Should remove SNAPSHOT

Comment (bdmogal) [reply dre-plugins/pom.xml]: if it works for you as is, perhaps try to remove the SNAPSHOT version from your local m2

**Review by bdmogal — CHANGES_REQUESTED** [review]
1 comment, rest LGTM

**Review by bdmogal — APPROVED** [review]

**Review by bdmogal — APPROVED** [review]
@CuriousVini added icon. please check. lgtm.

**Comment by CuriousVini** [comment]
Tested on sandbox, squashing and merging:

image

**Pull request merged**